src/test/java/org/jabref/logic/citationkeypattern/CitationKeyGeneratorTest.java

```diff
@@ -747,6 +747,9 @@ void testLastPage() {
         assertEquals("97", CitationKeyGenerator.lastPage("7,41,73--97"));
         assertEquals("97", CitationKeyGenerator.lastPage("7,41,97--73"));
         assertEquals("43", CitationKeyGenerator.lastPage("43+"));
+        assertEquals("0", CitationKeyGenerator.lastPage("00--0"));
+        assertEquals("1", CitationKeyGenerator.lastPage("1--1"));
+
```

Comment:
Please no empty line at the end of the method

Reply:
Please convert the whole test to a `@ParameterizedTest`. See https://github.com/JabRef/jabref/pull/7544 for an inspiration